src/user/using-the-capture-app.md

```diff
@@ -878,14 +878,14 @@ On the enrollment dashboard, you can view the Indicator widget.
 
 ![](resources/images/capture_app/enrollment-dash-indicator-widget-1.png)
 
-The indicator widget displays indicators from either program rules or program indicators. These indicators is a combination of keys and values.
+The indicator widget displays program indicators that are configured as "Display in form" or through program rules (Displaying text or key/value pairs in "Program indicator widget"). 
 
 #### Empty state
 If there are no indicators available for the enrollment, the widget will be shown with a text saying there are no indicators.
 If there are no program rules or indicators that can possibly display a message, the indicator will not be displayed at all.
```

Comment:
Do you mean the indicator **widget** will not be displayed at all?

For example:
"If there are no data available for indicators related to the enrollment, the widget will be shown with a text saying there are no indicators.
If there are no program rules or program indicators configured to be displayed in the Indicator widget, the widget will not be displayed at all."